index.html

```diff
@@ -55,7 +55,7 @@ <h2>第２回Git勉強会</h2>
                     <article>
                         <h3>イベント日時・場所</h3>
                         <p>3月23日 19:00開始</p>
-                        <p>株式会社〇〇 イベントセミナー会場</p>
+                        <p>株式会社ＭＫＡ イベントセミナー会場</p>
```

Comment:
### Just to be sure, could you tell me the below?
- [ ] Is it an official corporate name?

Reply:
### I've confirmed.
- [x] It's OK!

Reply:
### Thanks!
- Everything is OK!